doc/source/getting_started/learning.rst

```diff
@@ -38,11 +38,12 @@ Downloads
   These are versions valid during a year and with limited capabilities 
   regarding number of nodes, elements, etc.
 
-- View and download :download:`PyMAPDL cheatsheet <Cheat_Sheet_PyMAPDL.pdf>` to help
-  you to learn PyMAPDL.
+.. jinja:: cheat_sheet
+  
+  - View and download :download:`PyMAPDL cheatsheet <https://mapdl.docs.ansys.com/version/{{ version }}/_static/cheatsheet.pdf>`
+    to help you to learn PyMAPDL.
```

Comment:
I see no difference after using the `.. jinja::` directive:

<img width="934" alt="image" src="https://github.com/user-attachments/assets/69c6f9c3-6dc2-434b-b56e-3e445e314405">

Reply:
The `jinja` directive ensures the referred `cheat_sheet` corresponds to the correct PyMAPDL version.

Reply:
Ohh.. that's very clever!

Reply:
The jinja context was initiated by @Revathyvenugopal162. Thank you again!